src/executables/utility.h

```diff
@@ -587,7 +587,8 @@ typedef enum
   APPLYINFO_MSG_NOT_HA_MODE = 22,
   APPLYINFO_MSG_HA_NOT_SUPPORT = 58,
   APPLYINFO_MSG_NOT_IN_STANDALONE = 59,
-  APPLYINFO_MSG_USAGE = 60
+  APPLYINFO_MSG_USAGE = 60,
+  APPLYINFO_MSG_QUERY_RESULTS_FAIL = 117
```

Comment:
```suggestion
   APPLYINFO_MSG_HA_NOT_SUPPORT = 57,
   APPLYINFO_MSG_NOT_IN_STANDALONE = 58,
   APPLYINFO_MSG_NO_QUERY_RESULTS = 59,
   APPLYINFO_MSG_USAGE = 60
```

how about reordering the number? 
and changing `QUERY_RESULTS_FAIL` to `NO_QUERY_RESULTS` ?

Reply:
Thank you for your suggestion! I will apply it!